<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabServerWidget</class>
 <widget class="QWidget" name="TabServerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="grpBoxServerEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>241</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>服务器编辑</string>
   </property>
   <widget class="QLabel" name="labelA0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP地址:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelA1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>服务端口号:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineIpAddr">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineServerPort">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>26123</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineCtrlPort">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>20001</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelA5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>编辑操作:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineCtrlPwd">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="labelA4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>控制密码:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelA3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>监控周期:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxMonitorPeriod">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::UpDownArrows</enum>
    </property>
    <property name="specialValueText">
     <string/>
    </property>
    <property name="suffix">
     <string>秒</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>3600</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelA2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>控制端口号:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSrvAdd">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSrvModify">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>修改</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSrvDelete">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSrvStartMonitor">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>开始监控</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSrvStopMonitor">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>停止监控</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSrvClearList">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpBoxUpdate">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>240</y>
     <width>241</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>程序升级</string>
   </property>
   <widget class="QLabel" name="labelC0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>正在升级:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelC1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>更新文件:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineUpdateFilePath">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUpdate">
    <property name="geometry">
     <rect>
      <x>147</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>升级</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelC2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>更新状态:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnScanFile">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReset">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>重置</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineUpdateState">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineUpdatingServer">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpBoxRtState">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>240</y>
     <width>241</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>webhelper实时状态</string>
   </property>
   <widget class="QLabel" name="labelB5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>发送数据:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineSendSize">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineRecvSize">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelB6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>接收数据:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelB0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>线程池大小:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineThreadPoolSize">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineWorkThreadCount">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelB1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>工作线程数:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineRunTime">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelB7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>运行时间:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineThreadPoolLoad">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelB2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>线程池负载:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineUnGetResultNum">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelB3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>未获取结果:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineResultCacheCount">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelB4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>结果缓存数:</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grpBoxServer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>761</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>服务器列表</string>
   </property>
   <widget class="QTableWidget" name="tableServer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>741</width>
      <height>181</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideRight</enum>
    </property>
    <property name="showGrid">
     <bool>true</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::SolidLine</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>60</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>21</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>30</number>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>21</number>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>IP地址</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>服务端口</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>控制端口</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>运行状态</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>执行任务</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>任务总数</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>访问总数</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>监控周期</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>监控状态</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>版本信息</string>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tableServer</tabstop>
  <tabstop>lineIpAddr</tabstop>
  <tabstop>lineServerPort</tabstop>
  <tabstop>lineCtrlPort</tabstop>
  <tabstop>spinBoxMonitorPeriod</tabstop>
  <tabstop>lineCtrlPwd</tabstop>
  <tabstop>btnSrvAdd</tabstop>
  <tabstop>btnSrvModify</tabstop>
  <tabstop>btnSrvDelete</tabstop>
  <tabstop>btnSrvClearList</tabstop>
  <tabstop>btnSrvStartMonitor</tabstop>
  <tabstop>btnSrvStopMonitor</tabstop>
  <tabstop>lineThreadPoolSize</tabstop>
  <tabstop>lineWorkThreadCount</tabstop>
  <tabstop>lineThreadPoolLoad</tabstop>
  <tabstop>lineUnGetResultNum</tabstop>
  <tabstop>lineSendSize</tabstop>
  <tabstop>lineRecvSize</tabstop>
  <tabstop>lineRunTime</tabstop>
  <tabstop>lineUpdateFilePath</tabstop>
  <tabstop>lineUpdateState</tabstop>
  <tabstop>btnScanFile</tabstop>
  <tabstop>btnUpdate</tabstop>
  <tabstop>btnReset</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
